### PR TITLE
added xml parser and API request for geocoding

### PR DESCRIPTION
**Pull request by zhibekbastian**

### CODE REVIEW

File: Backend/Core/APIServer.py [hunk 1]
```diff
@@ -42,6 +43,9 @@ def _checkData(self, requestName, data):
                 "latitude",
                 "longitude",
                 "number"
+            ],
+            "GeocodingAddress": [
+                "q"
```
Comment (lkno0705) [inline Backend/Core/APIServer.py:48]: What is "q" ? What does it stand for? A param which is just called q isn't really meaningfull.

Comment (zhibekbastian) [reply Backend/Core/APIServer.py:48]: q stands for query, it’s from the Geocoding API, free form query string to search for.

Comment (zhibekbastian) [reply Backend/Core/APIServer.py:48]: > What is "q" ? What does it stand for? A param which is just called q isn't really meaningfull.

q stands for query, it’s from the Geocoding API, free form query string to search for.

**Comment by zhibekbastian** [comment]
> Also the documentation for this request is missing in our Wiki.

done

**Pull request merged**